cmd/snap-failure/cmd_snapd_test.go

```diff
@@ -35,3 +41,171 @@ func (r *failureSuite) TestRun(c *C) {
 	c.Check(err, IsNil)
 	c.Check(r.Stderr(), HasLen, 0)
 }
+
+func writeSeqFile(c *C, name string, current snap.Revision, seq []*snap.SideInfo) {
+	seqPath := filepath.Join(dirs.SnapSeqDir, name+".json")
+
+	err := os.MkdirAll(dirs.SnapSeqDir, 0755)
+	c.Assert(err, IsNil)
+
+	b, err := json.Marshal(&struct {
+		Sequence []*snap.SideInfo `json:"sequence"`
+		Current  string           `json:"current"`
+	}{
+		Sequence: seq,
+		Current:  current.String(),
+	})
+	c.Assert(err, IsNil)
+
+	err = ioutil.WriteFile(seqPath, b, 0644)
+	c.Assert(err, IsNil)
+}
+
+func mockCommandInDir(c *C, path, script string) *testutil.MockCmd {
```

Comment:
We should probably tweak testutil.MockCommand to create the dir if needed, this is a recurring pattern in tests (not for this PR).